easydita-knowledge-portal/functions.php

```diff
@@ -148,9 +148,9 @@ function easydita_knowledge_portal_get_hierarchy() {
 		$hierarchy = array();
 		if($post->ID) {
 			$hierarchy[] = $post->ID;
-			
```

Comment:
Huh, interesting. Ok, well maybe we can figure out how to avoid this in the future, but no worries.